### PR TITLE
Bug 1625965 - Raptor Youtube-Playback Fenix tests are silently failin…

### PR DESCRIPTION
**Pull request by bebef1987**
…g on VP9

### CODE REVIEW

File: 2019/harness/test.js [hunk 1]
```diff
@@ -311,6 +311,8 @@ TestExecutor.prototype.onfinished = function() {
           switch (harnessConfig.testType) {
             case "playbackperf-test":
               results[test.prototype.desc] = {
+                passes: test.prototype.passes,
+                fail: test.prototype.failures,
```
Comment (whimboo) [inline 2019/harness/test.js:319]: Should whether be `pass` or `failures`.

**Review by whimboo — CHANGES_REQUESTED** [review]
Can you please explain why those changes are necessary?

File: 2019/harness/test.js (hunk 1 above)
Comment (bebef1987) [reply 2019/harness/test.js:319]: Used `fail`

File: 2019/media/playbackperfTest.js [hunk 2]
```diff
@@ -204,7 +204,14 @@ var createPlaybackPerfTest = function(
...
           runner.fail('UserAgent was unable to render any frames.');
         }
-        runner.checkLE(totalDroppedFrames, 1, 'Total dropped frames');
+        if (runner.isRaptorTest()){
+          if (totalDroppedFrames > 1) {
+            runner.log('Total dropped frames is (' + totalDroppedFrames +
+                ') which should be less than or equal to (1)');
+          }
+        } else {
+          runner.checkLE(totalDroppedFrames, 1,);
```
Comment (gmierz) [inline 2019/media/playbackperfTest.js]: Missing `'Total dropped frames'` as the last parameter.

**Review by gmierz — APPROVED** [review]
This looks good to me, thanks!

**Review by whimboo — APPROVED** [review]

**Comment by whimboo** [comment]
Given that Greg is fine with those changes lets get it merged.

**Comment by whimboo** [comment]
@marianrai does our dev server for the 2020 work already got this change?

**Pull request merged**